templates/takeovers/_1904-takeover.html

```diff
@@ -0,0 +1,90 @@
+        <p class="p-heading--four u-no-margin--right">
+          From the Linux Kernel to Openstack and Kubernetes: find out why the
+          latest Ubuntu release is our most advanced to date.
+        </p>
+        <ul class="p-inline-list u-hide--small">
+          <li class="p-inline-list__item">
+            <a
+              href="/download"
+              onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : '19.04 takeover', 'eventLabel' : 'What’s new in Ubuntu 19.04?', 'eventValue' : undefined });"
+              class="p-button--neutral"
+              ><span>Download Ubuntu</span></a
+            >
+          </li>
+          <li class="p-inline-list__item">
+            <a
+              href="https://www.brighttalk.com/webcast/6793/354804?utm_source=Takeover&utm_medium=Takeover&utm_campaign=CY19_DC_19.04_WBN_19.04"
+              onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : '19.04 takeover', 'eventLabel' : 'Register for the webinar', 'eventValue' : undefined });"
+              class="p-link--inverted"
+              >Register for the webinar&nbsp;›</a
+            >
+          </li>
+        </ul>
+      </div>
+      <div class="col-4">
+        <p class="p-takeover__image u-align-text--center">
+          <img
+            src="{{ ASSET_SERVER_URL }}a76865fa-disco_dingo_mascot.svg"
+            alt=""
+          />
+        </p>
+        <p class="u-hide--medium u-hide--large">
+          <a
+            href="/download"
+            onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : '19.04 takeover', 'eventLabel' : 'What’s new in Ubuntu 19.04?', 'eventValue' : undefined });"
+            class="p-button--neutral"
+            ><span>Download Ubuntu</span></a
+          >
+        </p>
+        <p class="u-hide--medium u-hide--large">
+          <a
+            href="https://www.brighttalk.com/webcast/6793/354804?utm_source=Takeover&utm_medium=Takeover&utm_campaign=CY19_DC_19.04_WBN_19.04"
+            onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : '19.04 takeover', 'eventLabel' : 'Register for the webinar', 'eventValue' : undefined });"
+            class="p-link--inverted"
+            >Register for the webinar&nbsp;›</a
+          >
+        </p>
+      </div>
+    </div>
+  </div>
+  <style>
+    .p-takeover {
+      background-image: url("{{ ASSET_SERVER_URL }}8951dc67-suru-left.svg"),
+        url("{{ ASSET_SERVER_URL }}55079ac4-suru-right.svg"),
+        linear-gradient(
+          0deg,
+          #dd4814 0%,
+          #be3d2f 23%,
+          #76236d 75%,
+          #531a4d 100%
+        );
+      background-position: left, right;
+      background-repeat: no-repeat, no-repeat;
+      background-size: auto 100%, auto 100%, cover;
+    }
+
+    .p-takeover__image {
```

Comment:
You should set the overlay on larger then `620px` as the issue only effects small screens